### PR TITLE
Khdevops training

### PR DESCRIPTION
**Pull request by Official-simple-soul**
image
image
image

### CODE REVIEW

File: src/app/training/page.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            DevOps expert
+          </p>
+          <a
+            href="https://paystack.com/pay/khdevops"
+            target="_blank"
+            className="flex justify-start"
+          >
+            <button className="py-3 px-8 rounded text-white bg-pri">
+              Sign Up Now
+            </button>
```
Comment (aniebiet-udoh) [inline src/app/training/page.tsx:37]: This button should link to paystack payment page.

File: src/data/trainingData.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      {
+        img: '/images/cost-aws.svg',
+        title: 'Cost',
+        items: ['$250 per participant'],
+      },
+    ],
+  },
+  {
+    id: 'azure',
+    title: 'Azure Training',
```
Comment (aniebiet-udoh) [inline src/data/trainingData.js]: Change to 'Azure Cloud Training'

File: src/data/trainingData.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    paragraph: `Azure, Microsoft's cloud computing platform, provides a comprehensive suite of services for building, deploying, and managing applications. The Azure training is targeted at a demographic consisting mostly of newcomers to the Cloud DevOps space who want to immediately acquire useful knowledge that places them in a position to get entry-level jobs in DevOps. Ultimately, the goal is for participants to acquire the AZ900 certification after completing the course.`,
```
Comment (aniebiet-udoh) [inline src/data/trainingData.js]: See what ChatGPT suggested: ```Jumpstart your career in cloud computing with our Azure training! Designed for newcomers to Cloud DevOps, this course provides essential, hands-on skills to help you land entry-level DevOps roles. Complete the program and earn the AZ900 certification, positioning you for success in this rapidly growing tech field.```

File: src/data/trainingData.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+export const trainingData = [
+  {
+    id: 'aws',
+    title: 'AWS Cloud Services',
```
Comment (aniebiet-udoh) [inline src/data/trainingData.js]: Change this to 'AWS Cloud Training`

File: src/data/trainingData.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    paragraph: `Amazon Web Services provides a comprehensive suite of services for building, deploying, and managing applications. The AWS training is targeted at a demographic consisting mostly of newcomers who want to immediately acquire useful knowledge that will give them entry-level jobs as AWS cloud practitioners. The goal is for participants to acquire the AWS CCP (Certified Cloud Practitioner) certification after completing the course.`,
```
Comment (aniebiet-udoh) [inline src/data/trainingData.js]: ChatGPT suggestion: ```Launch your career in cloud computing with our AWS training! Perfect for newcomers, this course offers hands-on knowledge to help you secure entry-level roles as an AWS Cloud Practitioner. Finish the program and earn your AWS CCP (Certified Cloud Practitioner) certification, setting you up for success in the cloud industry.```

File: src/app/training/page.tsx (hunk 1 above)
Comment (Official-simple-soul) [reply src/app/training/page.tsx:37]: It does

Comment (Official-simple-soul) [reply src/app/training/page.tsx:37]: There is an anchor tag wrapped around it

File: src/data/trainingData.js (hunk 5 above)
Comment (Official-simple-soul) [reply src/data/trainingData.js]: updated

File: src/data/trainingData.js (hunk 4 above)
Comment (Official-simple-soul) [reply src/data/trainingData.js]: updated

File: src/data/trainingData.js (hunk 3 above)
Comment (Official-simple-soul) [reply src/data/trainingData.js]: updated

File: src/data/trainingData.js (hunk 2 above)
Comment (Official-simple-soul) [reply src/data/trainingData.js]: updated

File: src/app/training/page.tsx (hunk 1 above)
Comment (Official-simple-soul) [reply src/app/training/page.tsx:37]: updated

**Pull request merged**